evergreen/convert-client-cert-to-pkcs12.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with an error if any of the commands fail
+
+# Environment variables used as input:
+#   CLIENT_PEM                      Path to mongo -orchestration's client.pem: must be set.
+#   MONGO_X509_CLIENT_P12           filename for client certificate in p12 format
+#
+# Enviornment variables produced as output:
+#   MONGODB_X509_CLIENT_P12_PATH    Absolute path to client certificate in p12 format
+#   MONGO_X509_CLIENT_P12_PASSWORD  password for client certificate
+
+
+CLIENT_PEM=${CLIENT_PEM:-nil}
+MONGO_X509_CLIENT_P12=${MONGO_X509_CLIENT_P12:-client.p12}
```

Comment:
Alternative names could be `MONGO_X509_CLIENT_CERTIFICATE_PATH` and `MONGO_X509_CLIENT_CERTIFICATE_PASSWORD`

Reply:
What does P12 mean? The `Alternative names` sound more understandable to me. P12 confuses me a bit.